hello-hwasan/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
Do we have a higher level `.gitignore` that's catching things like `app/build` and `app/.cxx`?